src/components/ApparatusData.js

```diff
@@ -31,7 +33,18 @@ export default function ApparatusData({ filteredVariantObjects }) {
                     } else {
                       return (
                         <div key={`${i}-source`} className='block ml-2'>
-                          {currentSource.title}
+                          {
+                            currentSource.title.split(' ').map(
+                              (src) => (
+                                <span className="tooltip mr-1">
```

Comment:
Needs a key. Change to:

`                                <span key={`${i}-source-${j}-title`} className="tooltip mr-1">`

Reply:
Also, add `, j` to the map params on line 38